util/client.go

```diff
@@ -0,0 +1,12 @@
+package util
+
+import (
+	"os"
+
+	"gopkg.in/resty.v1"
+)
+
+// CreateClient - Returns a new configured API Client
+func CreateClient() *resty.Request {
```

Comment:
This should be declared in its' corresponding layer, as in clear architecture one of it's principles is to have the logic on it's corresponding layer. This could be in a repository.

Reply:
Done in the next deliverable!